llm/cli/main.py

```
@@ -68,8 +68,29 @@ def build_parser() -> argparse.ArgumentParser:
     parser.add_argument("--config", required=True, help="Path to the user-written JSON config file.")
     parser.add_argument("--prompt", help="Prompt text for one-shot generation.")
     parser.add_argument("--interactive", action="store_true", help="Read prompts interactively after loading the model.")
+    parser.add_argument("--serve", action="store_true", help="Start HTTP serving mode with OpenAI-compatible API.")
+    parser.add_argument("--host", default="0.0.0.0", help="Host to bind the serving server (default: 0.0.0.0).")
```

Ruff (0.15.12)

[error] Possible binding to all interfaces

(S104)